### ISSUE TITLE
TrustMate Integration API support

### ISSUE THREAD

**Issue by adamwaclawczyk** [TrustMateSA/MagentoPlugin#28]
https://github.com/TrustMateSA/MagentoPlugin/blob/1fa5cf8d0e4d452727776f055d360c552dbfe195/Enum/TrustMateApiEnum.php#L13

I can see that this extension is using the deprecated public API endpoints https://trustmate.io/public/documentation

Are there any plans for adding support for Integration API? https://trustmate.io/integration/documentation/v1

The integration doc page states that
> If you plan to show reviews using other method like TrustMate widgets or Display API you'll probably only need to use single endpoint from here: POST /public/api/invitation to create review inviations from shop orders

So I assume it is fine but, are there any drawbacks to using this extension which uses the public api - even if it is deprecated?

**Comment by zergu** [comment]
That's true but no worries, we'll be upgrading to newest API in next releases and we control both Magento module and API so it won't stop working without proper warnings. Also this deprecation is quite recent, this API will probably work as least one year, probably more.
